src/main/java/se/sundsvall/casedata/service/MessageService.java

```diff
@@ -28,19 +25,24 @@
 @Service
 public class MessageService {
 
+	private static final String TYPE = "UPDATE";
+	private static final String DESCRIPTION = "Meddelande mottaget";
```

Comment:
Prefix these two constants with "NOTIFICATION" for better understanding of the purpose?
ie. 
```
	private static final String NOTIFICATION_TYPE = "UPDATE";
	private static final String NOTIFICATION_DESCRIPTION = "Meddelande mottaget";
```